udata_front/theme/gouvfr/assets/js/pages/OrganizationPublishingForm/Step1CreateOrJoinOrganization.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div class="fr-container">
+    <Stepper :steps="props.steps" :currentStep="0"/>
+    <Container>
+      <ActionCard
+        :title="$t('Join an Organization')"
+        :content="$t('An organization is an entity in which many users can collaborate. The published datasets on the organization\'s name can be edited by its members', {site: title})"
+        :icon="schemaIcon"
+        :stretchHeight="true"
+      >
+      <template #actions>
+        <div class="fr-col-12 fr-grid-row fr-grid-row--middle">
+          <div class="fr-col-12 fr-col-lg-9 fr-pb-2v">
+            <MultiSelect
+              :searchPlaceholder="$t('Search an organization...')"
+              :allOption="$t('All organizations')"
+              suggestUrl="/organizations/suggest/"
+              entityUrl="/organizations/"
+              values=""
+            />
+          </div>
+          <div class="fr-col-12 fr-col-lg-3 fr-grid-row fr-grid-row--right">
+            <p class="fr-text--sm fr-px-1v fr-col-auto">{{ $t('or') }}</p>
+            <button class="fr-btn fr-col-auto" @click="$emit('start')">
+              {{ $t("Create an organization") }}
+            </button>
```

Comment:
The margin is broken here? 
![image](https://github.com/etalab/udata-front/assets/1770543/810a9961-7e0a-4f81-9e43-2090eab2e061)